datafusion/physical-expr/src/functions.rs

```diff
@@ -1369,9 +1369,9 @@ mod tests {
                 lit("abcde"),
                 lit(ScalarValue::Int8(Some(2))),
             ],
-            Err(DataFusionError::Internal(
-                "function left requires compilation with feature flag: unicode_expressions.".to_string()
-            )),
+            internal_err!(
```

Comment:
This wasn't caused by this PR, but it almost seems like these should be `plan_err!` rather than `internal_err!` (though maybe they are internal because this should have been checked earlier